libobs/obs-output.c

```diff
@@ -978,46 +1038,69 @@ void obs_output_set_preferred_size(obs_output_t *output, uint32_t width,
 		return;
 	}
 
-	output->scaled_width = width;
-	output->scaled_height = height;
+	// Only really matters for non-encoded
```

Comment:
I would move this to the `if` statement below, only changing it if the output is raw, assuming these parameters only affect raw outputs.